samples/dotnet/ConsoleConsumer/Program.cs

```diff
@@ -156,15 +156,15 @@ private static void StartPageViewConsumer(ConsumerConfig conf, CancellationToken
                 .SetPartitionsAssignedHandler(LogAssignedPartitions)
                 .SetPartitionsRevokedHandler(LogRekovedPartitions);
 
-            var avroSchema = PageViews._SCHEMA;
-            var schemaRegistry = new LocalSchemaRegistry(avroSchema.ToString());
-            if (avroSchema != null)
-            {
-                // if (typeof(TKey) != typeof(Ignore))
-                //     builder.SetKeyDeserializer(new MagicAvroDeserializer<TKey>(new AvroDeserializer<TKey>(schemaRegistry)));
+            //var avroSchema = PageViews._SCHEMA;
```

Comment:
nit: we should remove the code we don't need so that it is less confusing to people reading. Unless the comments provide a different way to use the sample.

Reply:
Done